Templates/Project/MainWindow.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<Window x:Class="MaterailDesign.WpfApp.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:mdix"
+        xmlns:materialDesign="http://materialdesigninxaml.net/winfx/xaml/themes"
+        mc:Ignorable="d"
+        TextElement.Foreground="{DynamicResource MaterialDesignBody}"
+        TextElement.FontWeight="Regular"
+        TextElement.FontSize="13"
+        TextOptions.TextFormattingMode="Ideal" 
+        TextOptions.TextRenderingMode="Auto"
+        Background="{DynamicResource MaterialDesignPaper}"
+        FontFamily="{materialDesign:MaterialDesignFont}" 
```

Comment:
Mabye use the new `Style="{StaticResource MaterialDesignWindow}"` from https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2174 instead?

Reply:
I was considering that, but it would have to be conditional based on the NuGet package version. But probably still work doing.